MapboxCoreNavigation/MGLMapboxEvents.swift

```diff
@@ -0,0 +1,39 @@
+import Mapbox
+import Polyline
+
+extension MGLMapboxEvents {
+    class func addDefaultEvents(routeProgress: RouteProgress, sessionIdentifier: UUID) -> [String: Any] {
+        var modifiedEventDictionary: [String: Any] = [:]
+    
+        modifiedEventDictionary["platform"] = UIDevice.current.systemName
```

Comment:
We can wrap this in a compiler flag by platform and by default we can set this to `unknown`

E.g.
```swift
#if os(iOS)
        UIApplication.shared.isIdleTimerDisabled = true
#endif
```